docs/en/docs/contributing.md

```diff
@@ -259,7 +259,7 @@ $ uvicorn tutorial001:app --reload
 
 </div>
 
-as Uvicorn by default will use the port `8000`, the documentation on port `8008` won't clash.
+as Uvicorn by default will use the port `8000`, the documentation on port `8008` won't crash.
```

Comment:
Based on your comment I've investigated and it looks like the most known term seems to be `overlap`. :thinking: 
```suggestion
as Uvicorn by default will use the port `8000`, the documentation on port `8008` won't overlap.
```

Reply:
I agree with you. "overlap" is better than "crash"